process_pr.py

```diff
@@ -2053,6 +2053,26 @@ def process_pr(repo_config, gh, repo, issue, dryRun, cmsbuild_user=None, force=F
             devReleaseRelVal=devReleaseRelVal,
             branch=pr.base.ref,
         )
+        if "PULL_REQUESTS" in global_test_params:
+            messageFullySigned += "\n**Notice** This PR was tested with additional Pull Request(s), please also merge them if necessary: "
+            linked_prs = global_test_params["PULL_REQUEST"].split()
+            prepend_comma = False
+            if not dryRun:
+                for linked_pr in linked_prs[1:]:
+                    linked_pr_repo, linked_pr_id = linked_pr.split("#")
+                    r = gh.get_repository(linked_pr_repo)
+                    linked_pr = r.get_issue(linked_pr_id)
+                    if linked_pr.state != "closed":
+                        linked_pr.create_comment(
```

Comment:
@iarspider , only call to `create_comment` should be under `not dryRun` condition